rasa/shared/utils/schemas/events.py

```diff
@@ -0,0 +1,155 @@
+ENTITIES_SCHEMA = {
+    "type": "array",
+    "items": {
+        "type": "object",
+        "properties": {
```

Comment:
how making `entity` and `value` required properties?

Reply:
I wouldn't make them required at this schema level because it wouldn't be applicable for the `UserUttered` event because of [this.](https://github.com/RasaHQ/rasa/blob/d4d606b12370d440d9f9ed3f9e4d8e2a23b959b8/rasa/shared/core/events.py#L396). However I can make them required in the `EntitiesAdded` subschema, as that shouldn't be an empty list.

Reply:
Sorry - how is the linked code blocking making `entity` and `value` required properties? Any empty list would still satisfy the schema, no?

Reply:
You're absolutely right - I was under the false impression that required properties will make an empty list fail but just added a few test cases for empty list and they passed 👍